grpc/src/main/java/com/linecorp/armeria/client/grpc/GrpcClientBuilder.java

```diff
@@ -387,6 +392,9 @@ public <T> T build(Class<T> clientType) {
         if (!clientInterceptors.isEmpty()) {
             option(GrpcClientOptions.INTERCEPTORS.newValue(clientInterceptors));
```

Comment:
Not related to this PR but for consistency.
```suggestion
            option(INTERCEPTORS.newValue(clientInterceptors));
```